roles/ucp/tasks/main.yml

```diff
@@ -32,6 +32,10 @@
     - "{{ ucp_instance_id_file }}"
   when: node_name != ucp_bootstrap_node_name
 
+- name: copy the ucp license to the remote machine
+  copy: src={{ ucp_license_file }} dest={{ ucp_license_remote }}
```

Comment:
@mapuri UCP needs a license in all cases, a trial license at minimal and so I had made it mandatory. Do you think, making it optional would be better? If so, I will make the changes.